lib/actions/preload.coffee

```diff
@@ -50,9 +50,11 @@ getApplicationsWithSuccessfulBuilds = (resin, deviceType) ->
 				build.status == 'success'
 		applications
 
-selectApplication = (expectedError, resin, form, deviceType) ->
+selectApplication = (expectedError, resin, form, deviceType, applicationInfoSpinner) ->
```

Comment:
I'd either:
* Just require resin-cli-visuals here - it's cheap, clear, doesn't cause any problems
* Pull these functions out of this file into `utils`, in a separate module where you can do requires up front like a normal node module (which you can then import here from inside the action function). https://github.com/resin-io/resin-cli/blob/master/lib/utils/patterns.coffee is an example - it requires at the top-level, but that's not a problem because everywhere that requires it does so from inside an action.

Up to you which one, depends on whether you think moving these functions will be useful for other parts of the code/future changes.